src/competencies/management/commands/import_competencies.py

```
@@ -0,0 +1,50 @@
+            if options['set']:
+                competency_set = load_bundled_set(options['set'])
+            else:
+                path = Path(options['file'])
+                if not path.is_file():
+                    raise CommandError(f"File not found: {path}")
+                if path.suffix.lower() == '.csv':
+                    competency_set = parse_csv_set(path.read_bytes(), name=path.stem)
+                else:
+                    competency_set = parse_json_set(path.read_bytes())
+        except ValidationError as e:
+            raise CommandError('; '.join(e.messages))
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)